KVKCalendar/Classes/WeekViewCalendar.swift

```diff
@@ -79,7 +79,7 @@ final class WeekViewCalendar: UIView {
     
     func reloadData(events: [Event]) {
         data.events = events
-        timelineView.createTimelinePage(dates: visibleDates, events: events, selectedDate: data.date)
+        timelineView.createTimelinePage(dates: visibleDates, events: events, selectedDate: data.date, showVerticalLine: true)
```

Comment:
Maybe add a property to the `Style` to adda choice to users?

Reply:
Fixed in style